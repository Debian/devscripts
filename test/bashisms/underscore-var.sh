#!/bin/sh

echo BASHISM $_
