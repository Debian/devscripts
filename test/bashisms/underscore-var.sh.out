possible bashism in bashisms/underscore-var.sh line 3 ($_):
echo BASHISM $_
